double_entry.gemspec

```diff
@@ -17,6 +17,32 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
+  gem.post_install_message = <<-'POSTINSTALLMESSAGE'
+Please note the following changes in DoubleEntry:
+ - New table `double_entry_line_metadata` has been introduced and is required for
+   aggregate reporting filtering. Existing applications will have to manually
```

Comment:
I think we should require it and not have to manage the possibility that the table doesn't exist.  Maybe we should make this wording a bit more insistent sounding (it does sound a little bit like it's optional)